sdk/storage/azure-storage-common/CMakeLists.txt

```diff
@@ -52,6 +52,7 @@ set(
     inc/azure/storage/common/internal/file_io.hpp
     inc/azure/storage/common/internal/reliable_stream.hpp
     inc/azure/storage/common/internal/shared_key_policy.hpp
+    inc/azure/storage/common/internal/shared_key_policy_lite.hpp
```

Comment:
The filename should be `shared_key_lite_policy.hpp` right? `lite` decorates the authentication scheme rather than the policy.

Reply:
done